_pages/algorithm-development/ci-cd.md

```diff
@@ -15,6 +15,13 @@ Continuous Integration & Deployment are standard practice in the world of softwa
 
 Algorithmia supports deployment and redeployment via the [the Algorithmia API]({{site.baseurl}}/algorithm-development/algorithm-management), and this is easily integrated into CI/CD tools such as Jenkins or GitHub Actions. With Algorithmia and your favorite CI/CD tool, your models are deployed as soon as they are ready, and can be instantly redeployed whenever an approved retrained model is available.
 
-<a href="https://github.com/algorithmiaio/model-deployment/tree/master/jenkins_deploy_algorithmia" class="btn btn-default btn-primary"><i class="fa fa-github" aria-hidden="true"></i> TRY IT OUT: Jenkins</a>
+For setting up an automated workflow in your **Algoritm repository**, you can check out the following examples with Jenkins or Github Actions:
 
-<a href="https://github.com/algorithmiaio/model-deployment/tree/master/githubactions_deploy_algorithmia" class="btn btn-default btn-primary"><i class="fa fa-github" aria-hidden="true"></i> TRY IT OUT: GitHub Actions</a>
+<a href="https://github.com/algorithmiaio/githubactions-modeldeployment-deprecated/tree/master/jenkins_deploy_algorithmia" class="btn btn-default btn-primary"><i class="fa fa-github" aria-hidden="true"></i> TRY IT OUT: Jenkins</a>
+
+<a href="https://github.com/algorithmiaio/githubactions-modeldeployment-deprecated/tree/master/githubactions_deploy_algorithmia" class="btn btn-default btn-primary"><i class="fa fa-github" aria-hidden="true"></i> TRY IT OUT: GitHub Actions in Algorithm Repo</a>
+
+
+For setting up an automated workflow in your **machine learning repository at Github**, where you may be using a Jupyter notebook to train/test/export your ML model or checking-in your saved model; you can check out our Github Action that automates your model and algorithm code deployment once you push your changes to your model development repo:
```

Comment:
Instead of "Algorithm repository" vs. "machine learning repository at Github", would it make sense to use wording such as "using your Algorithmia-hosted Git repository" vs. "using your GitHub-hosted Git repository"?

Reply:
Hey Kris, this new Github Action isn't something you integrate with your algorithm repository (regardless of it being backed by Github or Algorithmia) It is something that you integrate into your repo, where you're iterating on your models, typically using Jupyter notebooks. So this is your Jupyter notebook's repo, independent of your algorithm repo. 

In the scenario where this Github Action makes sense, you have a separate repo containing your Jupyter notebook, this notebook creates/exports your ML model, and this Github Action is integrated into this repo. More info here at this link :) https://github.com/algorithmiaio/algorithmia-modeldeployment-action

Reply:
So with this information in mind, I'd be happy to hear your suggestion on the button texts. I'm not super proud of my naming either! :)

Reply:
Gotcha, that makes sense, thank you for clarifying!

How about:

> For setting up an automated workflow in the repository with your **Algorithm serving function**
> TRY IT OUT: Jenkins for Algorithm Serving Repo
> TRY IT OUT: GitHub Actions for Algorithm Serving Repo

> For setting up an automated workflow in your repository with your **training notebook/script**
> TRY IT OUT: GitHub Actions for Model Training Repo

Or some version of that.

Then there are people like me that use the same repo for both. 🤣 

Feel free to use/change any combination of my suggestions... or stick with yours!

Reply:
I like this much better than mine, happy to go with them - thanks a bunch for the feedback Kris!